compose/ui/ui/build.gradle

```diff
@@ -206,7 +206,8 @@ if(AndroidXComposePlugin.isMultiplatformEnabled(project)) {
                     api(project(":compose:ui:ui-text"))
                     api(libs.skikoCommon)
                     implementation(libs.atomicFu)
-                    implementation(project(":compose:ui:ui-backhandler"))
```

Comment:
This probably broke backward compatibility for klibs due to [KT-60874](https://youtrack.jetbrains.com/issue/KT-60874) [KLIB Resolve] Don't save transitive dependencies in manifest's `depends` property.

Please fix it before beta01, creating a stub project for `backhandler`

Reply:
```

* What went wrong:
Circular dependency between the following tasks:
:navigationevent:navigationevent-compose:compileDesktopMainJava
+--- :navigationevent:navigationevent-compose:compileKotlinDesktop
|    \--- :compose:ui:ui:desktopJar
|         +--- :compose:ui:ui:compileDesktopMainJava
|         |    +--- :compose:ui:ui:compileKotlinDesktop
|         |    |    \--- :compose:ui:ui-backhandler:desktopJar
|         |    |         +--- :compose:ui:ui-backhandler:compileDesktopMainJava
|         |    |         |    +--- :navigationevent:navigationevent-compose:desktopJar
```

Reply:
I'll try to use a non-compose library then

Reply:
https://github.com/JetBrains/compose-multiplatform-core/pull/2441